### ISSUE TITLE
Dataclass length attribute syntax error

### ISSUE THREAD

**Issue by alex-and-or** [brython-dev/brython#1788]
Hello,

The code below generates a syntax error because I'm using "length":

```
import dataclasses


@dataclasses.dataclass
class Foo:
    length: int
```

```
  File <string>, line 1
    def __init__(self,$$length:_type_$$length)->_return_type:
                      ^
SyntaxError: invalid syntax (unknown token [$]) 
```

It's easy enough to avoid using "length" as an attribute name but I'd figure I'd raise an issue in case there's an easy fix.

Thanks!

**Comment by PierreQuentel** [comment]
I could reproduce the same issue with versions until 3.9.5; there is a different error in 3.9.6, but the bug seems to be fixed in 3.10.0.

Can you confirm ?

**Comment by alex-and-or** [comment]
Yes, upgrading to 3.10.0 fixed the problem. Thanks!
